### PR TITLE
Add addresses stack

### PR DESCRIPTION
**Pull request by Thykof**
related to https://github.com/massalabs/massa-as-sdk/issues/90

### CODE REVIEW

**Review by damip — APPROVED** [review]

**Comment by damip** [comment]
Thanks !

File: web3-dev/smart-contracts/address-stack.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  * Autonomous SC:
+      A message sent at a moment when the stack was [A, B, C] and calling a
+      target function F of a smart contrtact D will yield the following stack
+      at the beginning of the execution of the target function:
+      `bottom [C, D] top`.
+      Note: C and D can be the same
+
+  * Local execution (not yet implemented, see https://github.com/massalabs/massa-sc-runtime/issues/170)
+      Local executions don't change the stack: they allow executing foreign code
```
Comment (longgamma-massa) [inline web3-dev/smart-contracts/address-stack.rst:33]: Local executions don't change the stack, they allow executing foreign code

File: web3-dev/smart-contracts/address-stack.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      in the current context
```
Comment (longgamma-massa) [inline web3-dev/smart-contracts/address-stack.rst:34]: .

File: web3-dev/smart-contracts/address-stack.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+ABIs
+----
+
+In the `massa assemblyscript sdk <https://github.com/massalabs/massa-as-sdk/>`_
+there is a Object called `Context`. You can import it in your smart contract
```
Comment (longgamma-massa) [inline web3-dev/smart-contracts/address-stack.rst:40]: there is an Object called `Context`. You can import it in your smart-contract code this way:

**Review by longgamma-massa — COMMENTED** [review]
some fixes of the Stack section suggested

**Comment by qdrn** [comment]
@Thykof do you mind making a followup PR with the suggestions of @longgamma-massa ?

**Comment by Thykof** [comment]
it has already been fixed in https://github.com/massalabs/massa-docs/pull/68/files

**Comment by qdrn** [comment]
Thanks!

**Pull request merged**